src/main.rs

```diff
@@ -355,3 +383,45 @@ fn error_404(req: &HttpRequest<crate::MiniserveConfig>) -> Result<HttpResponse,
         .into_string(),
     ))
 }
+
+// Print the QR code of the specified content in the terminal
+// For the width and height to be equal, two characters are used in each grid
+// Cannot control the line-height of the terminal, so it looks imperfect
+fn print_qrcode(content: String) {
```

Comment:
How about using something like this instead? https://github.com/svenstaro/qrcode-rust/blob/master/examples/encode_unicode.rs

Much less code for us and super clean unicode output.

Reply:
Looks good, should we use `qrcode` create?

Reply:
Yeah, then we should also change the current qrcodegen dependency to that.